config/master.d/master.conf

```diff
@@ -3,7 +3,8 @@ auto_accept: True
 interface: 0.0.0.0
 event_return: mysql
 presence_events: True
-worker_threads: 10
+worker_threads: 16
+timeout: 180
```

Comment:
I guess that's seconds, I think 3 minutes is really too long. What do the others think about it? Maybe 1 minute should be enough